python/paddle/fluid/optimizer.py

```diff
@@ -541,6 +501,18 @@ def _append_optimize_op(self, block, param_and_grad):
                                         param_and_grad[0])
         moment2 = self._get_accumulator(self._moment2_acc_str,
                                         param_and_grad[0])
+
```

Comment:
fixed in https://github.com/PaddlePaddle/Paddle/pull/12103

Reply:
#12103 is no good, because there is no need to add a `finish_update` method